models/streamitem.go

```diff
@@ -64,10 +64,11 @@ func (s *streamItemStore) GetItems(streamID snowflake.ID, startID snowflake.ID,
 		q = q.Where("item_id < ?", startID).Order("item_id DESC")
 	} else if direction == After {
 		q = q.Where("item_id > ?", startID).Order("item_id ASC")
+	} else {
+		panic("Bad direction")
 	}
-	q = q.Limit(count)
-	err := q.Find(&items).Error
-	if err != nil {
+
+	if err := q.Limit(count).Find(&items).Error; err != nil {
 		return nil, err
 	} else {
```

Comment:
No need for the `else`. There's very rarely a need for an `else`. There's never a need for an `if ... { return } else {`.